### DISCUSSION TITLE
Power button location

### DISCUSSION THREAD

**Discussion by JDZWOW** [valinet/ExplorerPatcher#1003] [Q&A]
Is there anyway for me to switch the power button location to the left and have the account details on the right?

**Comment by Poopooracoocoo** [comment]
I assume you're referring to the Windows 11 start menu. No, ExplorerPatcher does not let you change that.

**Reply by valinet** [reply]
\* Microsoft doesn't let you change that and ExplorerPatcher doesn't yet do anything about it.

**Comment by JDZWOW** [comment]
no I'm using the windows 10 setting not the windows 11.
when you click the start button you have a box that opens up and at the bottom there is the account icon on the left side and then on the right on the other side of the box there is the power button. Just looking for a way to reverse that.
doesn't feel right having the power button out towards the middle of the screen

**Reply by Poopooracoocoo** [reply]
That's confusing. Mind posting a screenshot?

**Reply by JDZWOW** [reply]
![1](https://user-images.githubusercontent.com/62466135/156907171-10b3f29a-0a51-405f-b261-8f063cf55f87.jpg)

see just was looking to find out if theres a way to swap the positions of that account icon and the power button

**Reply by Poopooracoocoo** [reply]
Yeah that's the Windows 11 start menu.

As said before, it's not possible to change that at the moment.

**Reply by JDZWOW** [reply]
hmmm  should it be different with this program? the start menu I mean.
I have it set on windows 10. Which does allow me to drag the taskbar to the left side of my screen unlike the windows 11 one.
Just wondering if I'm not actually seeing everything as it should be when set to windows 10

**Reply by Poopooracoocoo** [reply]
![image](https://user-images.githubusercontent.com/51363864/156908246-9bd9ac69-5a51-47d4-8c10-a6cab9f49c02.png)

**Reply by JDZWOW** [reply]
we must have a different version of this thing. I don't have all those options in that section
I have version 220000.469.41.16
I just downloaded this last night after I installed windows 11
what version do you have?

**Reply by Poopooracoocoo** [reply]
Sorry, I should've specified. My bad. I've got ExplorerPatcher pre-release 22000.493.42.29  and Windows 11 build 10.0.22000.527.

**Comment by JDZWOW** [comment]
oh ok so beta version?
I'll have to go look and redownload then

**Reply by Poopooracoocoo** [reply]
No need to redownload per se. ExplorerPatcher has a nice updater. Just go to the updates tab in EP's properties, enable pre-releases and check for updates! :3

**Comment by JDZWOW** [comment]
oh ok will do, thanks
